### ISSUE TITLE
Allow links to pending issues for no-suspicious-comment

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [Microsoft/tslint-microsoft-contrib#440]
Request: add an option to `no-suspicious-comment` to allow comments linking to an issue tracker.

This is bad:

```typescript
// TODO remove this when {work} happens
this.unsafeThingThatShouldBeRemovedSoon();
```

However, if we have a tracking item to resolve the TODO, it's useful to keep a link to that item in code:

```typescript
// See https://github.com/my-org/my-project/123
// TODO remove this when {work} happens
this.unsafeThingThatShouldBeRemovedSoon();
```

It could be configured like so:

```json
"no-suspicious-comment": [true, [
    "https://github.com/my-org/my-project/(.*)"
]],
```

...and if there are regexps allowed, change the rule warning string to mention the allowed formats?
